test/helpers/e2e.go

```diff
@@ -343,5 +368,48 @@ func newSimulatedEVML2SovereignChain(t *testing.T) (
 	require.NoError(t, err)
 	require.Equal(t, gerProxyAddr, bridgeGERAddr)
 
-	return client, setup.UserAuth, gerProxyAddr, gerL2Contract, setup.BridgeProxyAddr, setup.BridgeProxyContract
+	// Deploy AggOracleCommittee contract if EnableAggOracleCommittee is true
+	var aggOracleCommitteeProxyAddr common.Address
+	var aggOracleCommitteeContract *aggoraclecommittee.Aggoraclecommittee
+	if aggOracleCommitteeConfig.EnableAggOracleCommittee {
+		// Deploy AggOracleCommittee contract
+		aggOracleCommitteeAddr, _, _, err := aggoraclecommittee.DeployAggoraclecommittee(
+			setup.DeployerAuth, client.Client(), gerProxyAddr)
+		require.NoError(t, err)
+		client.Commit()
+
+		// Prepare initialize data that are going to be called by the aggoracle committee contract
+		aggOracleCommitteeAbi, err := aggoraclecommittee.AggoraclecommitteeMetaData.GetAbi()
+		require.NoError(t, err)
+		require.NotNil(t, aggOracleCommitteeAbi)
+
+		// aggOracleMembers are the addresses of the aggoracle committee members
+		aggOracleMembers := []common.Address{setup.UserAuth.From}
+
+		// add other aggoracle committee members to the aggOracleMembers slice based on the quorum
+		for i := 0; i < int(aggOracleCommitteeConfig.Quorum)-1; i++ {
+			aggOracleMembers = append(aggOracleMembers, crypto.CreateAddress(setup.DeployerAuth.From, aggOracleCommitteeNonce+uint64(i+1)))
+		}
+
+		aggOracleCommitteeInitData, err := aggOracleCommitteeAbi.Pack("initialize", setup.DeployerAuth.From, aggOracleMembers, aggOracleCommitteeConfig.Quorum)
+		require.NoError(t, err)
+
+		// Deploy a proxy contract for the aggoracle committee
+		aggOracleCommitteeProxyAddr, _, _, err = proxy.DeployProxy(
+			setup.DeployerAuth,
+			client.Client(),
+			aggOracleCommitteeAddr,
+			setup.DeployerAuth.From,
+			aggOracleCommitteeInitData,
+		)
+		require.NoError(t, err)
+		client.Commit()
+
+		// Create aggoracle committee contract binding
+		aggOracleCommitteeContract, err = aggoraclecommittee.NewAggoraclecommittee(
+			aggOracleCommitteeProxyAddr, client.Client())
+		require.NoError(t, err)
+	}
+
+	return client, setup.UserAuth, gerProxyAddr, gerL2Contract, setup.BridgeProxyAddr, setup.BridgeProxyContract, aggOracleCommitteeProxyAddr, aggOracleCommitteeContract
```

Comment:
Consider wrapping the line, as it is too long and linter in VSC complains

```suggestion
	return client, setup.UserAuth, gerProxyAddr, gerL2Contract,
		setup.BridgeProxyAddr, setup.BridgeProxyContract,
		aggOracleCommitteeProxyAddr, aggOracleCommitteeContract
```

Reply:
done